libraries/ESP8266WiFiMesh/src/ESP8266WiFiMesh.cpp

```diff
@@ -359,7 +359,7 @@ transmission_status_t ESP8266WiFiMesh::exchangeInfo(WiFiClient &currClient)
   verboseModePrint("Transmitting");  // Not storing strings in flash (via F()) to avoid performance impacts when using the string.
     
   currClient.print(getMessage() + '\r');
-  yield();
+  optimistic_yield(10000);
```

Comment:
A yield here ensures the message actually is sent over the wifi, no?  So making this an optimistic_yield doesn't give the OS time to call the LWIP stack in the SYS context and actually get the work done, it seems.

Reply:
rather err on the safe side, reverting.